ocs_ci/krkn_chaos/krkn_workload_factory.py

```
@@ -1041,12 +1041,18 @@ def get_blk_config():
+        num_cephfs_pvcs = self.config.get_num_cephfs_pvcs()
+        num_pvcs_by_interface = {
+            constants.CEPHBLOCKPOOL: num_rbd_pvcs,
+            constants.CEPHFILESYSTEM: num_cephfs_pvcs,
+        }
         pvc_size = self.config.get_pvc_size()
         use_encrypted = self.config.use_encrypted_pvc()
 
         log.info(
-            f"Creating {num_pvcs} PVCs per storage interface with size {pvc_size}Gi"
+            f"Creating {num_rbd_pvcs} RBD PVCs and {num_cephfs_pvcs} CephFS PVCs "
+            f"with size {pvc_size}Gi"
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)